packages/wayland_proxy_virtwl.rb

```diff
@@ -0,0 +1,143 @@
+require 'package'
+
+class Wayland_proxy_virtwl < Package
+  description 'Proxy Wayland connections across the VM boundary'
+  homepage 'https://github.com/talex5/wayland-proxy-virtwl'
+  @_ver = '69de5648bad98a0f7ae19bea292420d7fd804205'
+  version @_ver
+  compatibility 'all'
+  source_url 'https://github.com/talex5/wayland-proxy-virtwl.git'
+  git_hashtag @_ver
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/69de5648bad98a0f7ae19bea292420d7fd804205_armv7l/wayland_proxy_virtwl-69de5648bad98a0f7ae19bea292420d7fd804205-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/69de5648bad98a0f7ae19bea292420d7fd804205_armv7l/wayland_proxy_virtwl-69de5648bad98a0f7ae19bea292420d7fd804205-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/69de5648bad98a0f7ae19bea292420d7fd804205_i686/wayland_proxy_virtwl-69de5648bad98a0f7ae19bea292420d7fd804205-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/wayland_proxy_virtwl/69de5648bad98a0f7ae19bea292420d7fd804205_x86_64/wayland_proxy_virtwl-69de5648bad98a0f7ae19bea292420d7fd804205-chromeos-x86_64.tpxz'
+  })
+  binary_sha256({
+    aarch64: '8bc8e2d9520ad8928f135aefbe04a5099ed15230607fb66cea5c8ecbe796b4e9',
+     armv7l: '8bc8e2d9520ad8928f135aefbe04a5099ed15230607fb66cea5c8ecbe796b4e9',
+       i686: 'd936e559d1b9a77fd74d2f00353b80e574f67334ea238e9bd29cbb698cc70417',
+     x86_64: 'd6a34fb22a85eba404b45ff9a4ddc7351038104b8ef1719db37f5535eb401bee'
+  })
+
+  depends_on 'ocaml' => :build
+  depends_on 'opam' => :build
+  depends_on 'xwayland'
+
+  @OPAMROOT = "#{CREW_PREFIX}/share/opam"
+
+  def self.patch
+    # For more info see https://github.com/talex5/wayland-proxy-virtwl/issues/24
+    @xwayland_patch = <<~'PATCH_XWAYLAND_EOF'
+      --- a/wayland-proxy-virtwl/xwayland.ml     2021-11-18 20:34:27.000000000 +0000
+      +++ b/wayland-proxy-virtwl/xwayland.ml     2021-11-24 19:27:16.940505733 +0000
+      @@ -129,9 +129,7 @@ module Selection = struct
+           x11 : X11.Display.t Lwt.t;
```

Comment:
I’d more prefer using tools like `.sub` for patches, since it doesn’t need to re-generate a patch file after upstream updates (as the exact line number may be changed)

Reply:
How do I do that?

Reply:
(In this case, though, I'm hoping that there will be sufficient upstream changes in an eventual tagged release making these patches unnecessary.)

Reply:
Unlike sed, the `.sub` function accepts multi line strings as the replace target, even using a mutil line regex :) (with //m regex)